src/App.css

```diff
@@ -1,8 +1,13 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
 html,
 body {
     box-sizing: border-box;
     margin: 0;
     padding: 0;
+    background-color: #262626;
```

Comment:
This file could be converted to a `.scss` file, then you could import and use the global vars we created for `background-color` and `color` here.